Draft GSoC 2024 selection announcement (#335)
OpenRefine/openrefine.org: blog/2024-05-01-gsoc-intern.md

---
author: Antonin Delpeuch

> title: "Zyad joins us as a Google Summer of Code contributor"

Reviewer: ```suggestion
title: " Welcome to Zyad our 2024 Google Summer of Code intern"
```

Reviewer: adding the year in the title for easy reference later on. Follow the pattern started here https://openrefine.org/blog/2020/05/04/outreachy-gsoc-projects

Author: "Welcome to Zyad our 2024 Google Summer of Code intern" feels slightly un-grammatical, I would expect a comma after Zyad at least

Reviewer 2: Is the space before Welcome a typo?

> title: "Welcome to Zyad, our 2024 Google Summer of Code intern"

Author: Note: the official terminology from GSoC is "*contributor*", not "intern"

Reviewer 3: > Note: the official terminology from GSoC is "contributor", not "intern"

Correct, and they also don't use the term "internship," because that implies an employment relationship, which GSoC is not and they are careful to avoid any such implication.

---

> The OpenRefine project participates in the [Google Summer of Code](https://summerofcode.withgoogle.com/) program, which provides remote internships in software development on open source projects.

Reviewer: ```suggestion
The OpenRefine project participates in the 2024 [Google Summer of Code](https://summerofcode.withgoogle.com/) program, which provides remote internships in software development on open-source projects.
```

We are happy to announce that for this year's cohort, we will be joined by [Zyad Taha](https://github.com/zyadtaha), who has applied to work on improvements to clustering.
He will be mentored by [Antonin Delpeuch](https://github.com/wetneb).

> Please welcome Zyad in the community! Why don't you [introduce yourself to him](https://forum.openrefine.org/u/zyadtaha), explaining what you are doing in OpenRefine and why?

Reviewer: ```suggestion
Zyad already contributed several pull requests in the last week. However, this is the opportunity to welcome him into the community formally!  The Google Summer of Code internships will run from May 27 to August 26.
```

Reviewer: The call to action (introducing and explaining) didn't match the link to Zyad profile. I replaced it with the internship date aligning with the format of our previous announcement.
